File: beginner/python/1047.py
a,c,b,d=list(map(int,input().split()))

dif=((b*60)+d)-((a*60)+c)
if(dif<=0):
    dif+=24*60
    
time=dif//60
minute=dif%60
print("O JOGO DUROU %d HORA(S) E %d MINUTO(S)" % (time,minute))

File: beginner/javascript/1047.js
var input = require('fs').readFileSync('stdin', 'utf8');

var valores = input.split(' ');

var horaIni = parseInt(valores.shift());
var minIni = parseInt(valores.shift());
var horaFim = parseInt(valores.shift());
var minFim = parseInt(valores.shift());
var dif;

dif = ((horaFim * 60) + minFim) - ((horaIni * 60) + minIni);

if(dif <= 0) 
    dif += 24 * 60;

console.log("O JOGO DUROU " + parseInt(dif/60) + " HORA(S) E " + parseInt(dif%60) + " MINUTO(S)");    